src/include/rados/librados.hpp

```diff
@@ -303,7 +303,7 @@ namespace librados
     //flag mean ObjectOperationFlags
     void set_op_flags2(int flags);
 
-    void cmpext(uint64_t off, bufferlist& cmp_bl, int *prval);
+    void cmpext(uint64_t off, const bufferlist& cmp_bl, int *prval);
```

Comment:
Note: the backport PRs should leave the parameter non-const in previous releases.